pulp_ansible/app/modelresource.py

```diff
@@ -0,0 +1,97 @@
+from import_export import fields
+from import_export.widgets import ForeignKeyWidget, ManyToManyWidget
+from pulpcore.plugin.importexport import BaseContentResource, QueryModelResource
+from pulp_ansible.app.models import (
+    Role,
+    Collection,
+    Tag,
+    CollectionVersion,
+)
+
+
+class RoleContentResource(BaseContentResource):
+    """
+    Resource for import/export of ansible_role-content entities.
+    """
+
+    def set_up_queryset(self):
+        """
+        :return: Role content specific to a specified repo-version.
+        """
+        return Role.objects.filter(pk__in=self.repo_version.content)
+
+    class Meta:
+        model = Role
+        import_id_fields = model.natural_key_fields()
+
+
+class CollectionVersionContentResource(BaseContentResource):
+    """
+    Resource for import/export of ansible_collectionversion-content entities.
+    """
+
+    tags = fields.Field(
+        column_name="tags", attribute="tags", widget=ManyToManyWidget(Tag, field="name")
+    )
+    collection = fields.Field(
+        column_name="collection",
+        attribute="collection",
+        widget=ForeignKeyWidget(Collection, "name"),
```

Comment:
Thinking through this some: I don't think collection name is unique. I think we also need namespace.